### PR TITLE
TMCC-71: regressions fixes

### PR DESCRIPTION
**Pull request by rphh**

### CODE REVIEW

File: sdl2promela/translator.py [hunk 1]
```diff
@@ -1150,7 +1230,9 @@ def __generate_execute_transition(
             __build_assignment(
                 __generate_variable_name(context, target_variable_ref, True),
                 VariableReferenceBuilder(
-                    __get_parameter_name(input.parameters[0].target_variable)
+                    __get_parameter_name(
+                        parameters[0].target_variable
+                    )  # input.parameters[0].target_variable)
```
Comment (Lurkerpas) [inline sdl2promela/translator.py:1235]: Why leave this?

File: sdl2promela/translator.py [hunk 2]
```diff
@@ -1962,10 +2055,13 @@ def __generate_assignment(
 
     for name, dataType in finalType.Children.items():
         is_optional = dataType.Optional == "True"
-        if not is_optional and name not in right.elements:
-            raise Exception(
-                f"Invalid assignment to {finalType.CName}: missing required member {name}"
-            )
+        # candidates = [
```
Comment (Lurkerpas) [inline sdl2promela/translator.py:2058]: Why leave this?

File: sdl2promela/translator.py [hunk 3]
```diff
@@ -1962,10 +2055,13 @@ def __generate_assignment(
 
     for name, dataType in finalType.Children.items():
         is_optional = dataType.Optional == "True"
-        if not is_optional and name not in right.elements:
```
Comment (Lurkerpas) [inline sdl2promela/translator.py:1965]: Why remove this?

File: tests/integration/exported_procedure/resources/taste/work/controller/SDL/src/controller.pr [hunk 4]
```diff
@@ -0,0 +1,53 @@
+/* CIF PROCESS (250, 150), (150, 75) */
+process Controller;
```
Comment (Lurkerpas) [inline tests/integration/exported_procedure/resources/taste/work/controller/SDL/src/controller.pr:2]: We have two outputs, and we wait for both in any order. But, shouldn't the order be ensured? I assume that testing exported procedures with exported procedures could be a mess, BUT, if we are doing sporadic signals, then maybe the could carry a parameter that is increased in procedure and in transition, and very whether the correct value is received?

File: sdl2promela/translator.py (hunk 2 above)
Comment (rphh) [reply sdl2promela/translator.py]: This shall be uncommented, changed.

File: sdl2promela/translator.py (hunk 1 above)
Comment (rphh) [reply sdl2promela/translator.py]: Old comment, removed.

File: sdl2promela/translator.py (hunk 3 above)
Comment (rphh) [reply sdl2promela/translator.py:1965]: This is required, reverted

File: tests/integration/exported_procedure/resources/taste/work/controller/SDL/src/controller.pr (hunk 4 above)
Comment (rphh) [reply tests/integration/exported_procedure/resources/taste/work/controller/SDL/src/controller.pr:2]: changed, added parameter for procedure

**Review by Lurkerpas — APPROVED** [review]

**Pull request merged**